setup.py

```diff
@@ -29,7 +29,7 @@
         setup_requires=["incremental"],
         use_incremental=True,
         install_requires=[
-            "attrs",
+            "attrs>=17.1.0",
```

Comment:
Actually this should probably be 17.3.0 since that's when `auto_attribs` was introduced... https://www.attrs.org/en/stable/changelog.html?highlight=auto_attribs#id188

Reply:
Many thanks. for the update.
I think it should be 17.3.0 . As you mention, with 17.1.0 it will not work due to missing `auto_attribs` . It  would be interesting to run the test with minimum dependencies.

I don't know if there is a command-line option for pip for that.
I guess the other option is to have a separate minimum-deps build in which we pin the installed versions.